packages/dx-react-grid-bootstrap3/src/templates/filter-row/editor.test.jsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Editor } from './editor';
+
+const defaultProps = {
+  getMessage: key => key,
+};
+
+describe('Editor', () => {
+  it('should render a readonly input if disabled', () => {
+    const tree = shallow((
+      <Editor
+        {...defaultProps}
+        disabled
+      />
+    ));
+
+    expect(tree.find('input').prop('readOnly'))
+      .toBeTruthy();
+  });
+
+  it('should pass the className prop to the root element', () => {
+    const tree = shallow((
+      <Editor
+        {...defaultProps}
+        className="custom-class"
+      />
+    ));
+
+    expect(tree.is('.form-control'))
+      .toBeTruthy();
+    expect(tree.is('.custom-class'))
```

Comment:
You may combine this two `expect` into one. `...tree.is('.form-control.custom-class')`

Reply:
Yes, I know, thanks. But I prefer this way as more demonstrative